scripts/sct_label_vertebrae.py

```diff
@@ -133,13 +135,18 @@ def get_parser():
     parser.add_option(name="-param",
                       type_value=[[','], 'str'],
                       description="Advanced parameters. Assign value with \"=\"; Separate arguments with \",\"\n"
-                                  "shift_AP [mm]: AP shift of centerline for disc search. Default=" + str(param_default.shift_AP) + ".\n"
-                                  "size_AP [mm]: AP window size for disc search. Default=" + str(param_default.size_AP) + ".\n"
-                                  "size_RL [mm]: RL window size for disc search. Default=" + str(param_default.size_RL) + ".\n"
-                                  "size_IS [mm]: IS window size for disc search. Default=" + str(param_default.size_IS) + ".\n"
-                                  "gaussian_std [mm]: STD of the Gaussian function, centered at the most rostral point of the image, "
-                                  "and used to weight C2-C3 disk location finding towards the rostral portion of the FOV. Values to set "
-                                  "between 0.1 (strong weighting) and 999 (no weighting). Default=" + str(param_default.gaussian_std) + ".\n",
+                                  "shift_AP [mm]: AP shift of centerline for disc search. Default=" + str(
+                          param_default.shift_AP) + ".\n"
+                                                    "size_AP [mm]: AP window size for disc search. Default=" + str(
+                          param_default.size_AP) + ".\n"
+                                                   "size_RL [mm]: RL window size for disc search. Default=" + str(
+                          param_default.size_RL) + ".\n"
+                                                   "size_IS [mm]: IS window size for disc search. Default=" + str(
+                          param_default.size_IS) + ".\n"
+                                                   "gaussian_std [mm]: STD of the Gaussian function, centered at the most rostral point of the image, "
+                                                   "and used to weight C2-C3 disk location finding towards the rostral portion of the FOV. Values to set "
+                                                   "between 0.1 (strong weighting) and 999 (no weighting). Default=" + str(
+                          param_default.gaussian_std) + ".\n",
```

Comment:
I see it on github but not in Pycharm might be due to a rendering error I will check.

Reply:
I see what you mean is 6f55852 better ?

Reply:
Ah! I've just realized there is a conflict because of the update in #2842 (which replaced `msct_parser` with `argparse`).

The conflict affects this area specifically, so it's best to resolve that conflict, and then we can check back to these changes after.

Reply:
Merged with master done in 5c8279a

Reply:
Thanks @lrouhier! This nitpick is resolved now. :)